google/cloud/storage/examples/storage_bucket_samples.cc

```diff
@@ -936,14 +936,17 @@ void LockRetentionPolicy(google::cloud::storage::Client client, int& argc,
                 << ", status=" << original.status() << std::endl;
       return;
     }
-    StatusOr<void> status = client.LockBucketRetentionPolicy(
+
+    StatusOr<gcs::BucketMetadata> updated = client.LockBucketRetentionPolicy(
         bucket_name, original->metageneration());
-    if (!status.ok()) {
+    if (!updated.ok()) {
       std::cerr << "Error locking bucket retention policy for bucket "
-                << bucket_name << ", status=" << status.status() << std::endl;
+                << bucket_name << ", status=" << updated.status() << std::endl;
     }
+
     std::cout << "Retention policy successfully locked for bucket "
-              << bucket_name << std::endl;
+              << updated->name() << "\nFull metadata: " << *updated
```

Comment:
Add the `retentionPolicy.effectiveTime` to the output so it's not lost in the full metadata.

Reply:
That is a `std::chrono::time_point`, and, AFAIK, there is no formatting of those until C++20.  We have a formatting function in `google::cloud::storage::internal`, but really should not be using it in the examples.  I printed the retention policy in full, which is shorter than the full metadata anyway.

Reply:
I wonder if users will find it helpful to have a helper method that converts time_point to a human readable format, then again they can output all the metadata. 

I'll defer to community feedback on this case. Thanks for clarifying.

Reply:
If/when Abseil becomes a dependency we might consider using absl::Time instead of std::chrono::time_point, at which point this all gets much easier.